### PR TITLE
Fixed Intermission test

### PR DESCRIPTION
**Pull request by artrag**
All the changes are ok. I've integrated your latest version . NB: the 500/1000 tiles need a color attribute in the color array I've added a test for the "wonderful" intermission

### CODE REVIEW

File: mrdo_arcade.asm [hunk 1]
```diff
@@ -10550,7 +10707,7 @@ cvb_ANIMATEDLOGO:
 
 	CALL 	MYDISSCR
 	
-	LD		HL, $1900				; remove sprites in the new position of the SAT
+	LD		HL, PNT				; remove sprites in the new position of the SAT
```
Comment (artrag) [inline mrdo_arcade.asm:10710]: This should be SAT (or stay $1900)!!! Me bad sorry!

File: mrdo_arcade.asm [hunk 2]
```diff
@@ -11413,7 +11573,7 @@ MYMODE1:
 
 MYMODE2:
 	ld hl,mode
-	set 3,(hl)
+	set 7,(hl)			; intermission mode
```
Comment (artrag) [inline mrdo_arcade.asm:11576]: This was a bug on my side, it has to be bit 7 (not bit 3)

File: mrdo_arcade.asm [hunk 3]
```diff
@@ -8996,11 +9036,11 @@ MR_DO_PUSH_RIGHT_02_PATW:
 ;	DB 000,000,192,224,144,176,224,200
 ;	DB 248,056,128,128,128,000,000,000
 
-FIVE_HUNDRED_SCORE_PAT:
-   DB 000,000,000,000,000,113,066,114
-   DB 010,074,049,000,000,000,000,000
-   DB 000,000,000,000,000,140,082,082
-   DB 082,082,140,000,000,000,000,000
+;FIVE_HUNDRED_SCORE_PAT:
```
Comment (artrag) [inline mrdo_arcade.asm:9039]: If you keep my BONUS tiles loaded in the tileset, you can remove this pattern from the SPT 
My new tiles for the 500 and 1000 signs are at 208 and following

File: mrdo_arcade.asm [hunk 4]
```diff
@@ -8760,8 +8800,8 @@ old:
 	DB 005
 	DW BYTE_C294
 	DW MR_DO_PUSH_RIGHT_02_PAT
-	DB 000, 44*4, 000
-	DW FIVE_HUNDRED_SCORE_PAT
+;	DB 000, 44*4, 000
```
Comment (artrag) [inline mrdo_arcade.asm:8803]: If you keep my BONUS tiles loaded in the tileset, you can remove this from the sprite list

File: mrdo_arcade.asm [hunk 5]
```diff
@@ -8507,7 +8547,7 @@ PLAYFIELD_COLORS:
 	DW PHASE_10_COLORS
     ; CONTROL_BYTE,BLACK CHERRY STEM+BG, MED RED CHERRY+BG,.......BG1+BG2,BG Walls + Invisible Corridor)
 PHASE_01_COLORS: ; Med Green + Light Green
-	DB $00,$12,$82,$90,$80,$F0,$F0,$A0,$A0,$80,$23,$20,$C0,$C0,$80,$E0,$00,$00,$00,$00,$00,$00,$00,$00,$00,$00,$00,$F0,$F0,$F0,$F0,$F0
+	DB $00,$12,$82,$90,$80,$F0,$F0,$A0,$A0,$80,$23,$20,$C0,$C0,$80,$E0,$00,$00,$00,$00,$00,$00,$00,$00,$00,$00,$B1,$F0,$F0,$F0,$F0,$F0			; NOTE the $B1 is for the 500/1000 bonus in the tileset
```
Comment (artrag) [inline mrdo_arcade.asm:8550]: You can change the colors for bonus from $B1 to what you prefer

File: mrdo_arcade.asm (hunk 1 above)
Comment (plasticbugs) [reply mrdo_arcade.asm:10710]: No worries. I can fix when I merge!

**Pull request merged**